lib/shoryuken/manager.rb

```diff
@@ -241,5 +195,15 @@ def hard_shutdown_in(delay)
         end
       end
     end
+
+    def patch_batch!(sqs_msgs)
+      sqs_msgs.instance_eval do
+        def message_id
+          "batch-with-#{size}-messages"
+        end
```

Comment:
i have to check it out, it was part of fetcher before and i just moved it here. I'd like to get rid of that if possible. Will check if we need this (it think we did need it for logging).